libethcore/EthashSealEngine.cpp

```diff
@@ -54,7 +54,8 @@ void EthashSealEngine::generateSeal(BlockInfo const& _bi)
 	m_farm.setWork(m_sealing);
 	m_farm.start(m_sealer);
 	m_farm.setWork(m_sealing);		// TODO: take out one before or one after...
-	Ethash::ensurePrecomputed((unsigned)_bi.number());
+	if (option("precomputeDAG")[0] == 1)
```

Comment:
Since EthashSealEngine class does not guarantee that the option is set a check should be here.